charts/posit-chronicle/README.md.gotmpl

```diff
@@ -28,59 +29,101 @@ pod:
+Connect API key. The `extraObjects` value in the Connect chart can be used to
+create the secret and mount it to the Chronicle agent container. Due to the
+nature of the Chronicle agent, the pod may need to be restarted to pick up
+changes to the secret after initial deployment.
+
+## HTTPS Configuration
+
+Chronicle can be configured to use HTTPS for secure communication. The
+`config.HTTPS` section of the configuration allows you to specify the certificate
+and key files to use for HTTPS. Both `config.HTTPS.Certificate` and
+`config.HTTPS.Key` are expected to be paths to files accessible by Chronicle.
+The `extraSecretMounts` value can be used to mount the certificate and key files
+into the Chronicle pod. Here is an example of how to do this, assuming that
+the certificate and key files are stored together in a Kubernetes TLS secret:
+
+```yaml
+extraSecretMounts:
+  - name: chronicle-https
+    mountPath: /etc/chronicle/ssl
+    secretName: chronicle-https
+    items:
+      - key: tls.crt
+      - key: tls.key
+config:
+  HTTPS:
+    Enabled: true
+    Certificate: "/etc/chronicle/ssl/tls.crt"
+    Key: "/etc/chronicle/ssl/tls.key"
+```
 
 ## Storage Configuration
 
-Chronicle can be configured to persist data to a local Kubernetes Volume, AWS S3, or both.
+Chronicle can be configured to persist data to local storage, AWS S3, or both.
 
-The default configuration uses a local volume, which is suitable if you'd like to
-access and analyze the data within your cluster:
+The default configuration uses a local volume with persistence enabled, which
+is suitable if you'd like to access and analyze the data within your cluster:
 
 ```yaml
+persistence:
+  enabled: true
+  accessModes:
+    - ReadWriteOnce
+  size: 10Gi
 config:
   LocalStorage:
     Enabled: true
     Location: "/chronicle-data"
     RetentionPeriod: "30d"
 ```
 
-`retentionPeriod` controls how long usage data are kept. For example, `"120m"`
-for 120 minutes, `"36h"` for 36 hours, `14d` for two weeks, or `"0"` for unbounded retention.
-(Units smaller than seconds or larger than days are not supported.)
+The `persistence` section configures the persistent volume claim in the
+cluster while the `config.LocalStorage` section directly applies to Chronicle's
+configuration file. The persistent volume will always mount to the path specified
+by `config.LocalStorage.Path` to avoid potential misconfiguration and data loss.
+
+By default, Chronicle requests 10Gi of storage. In most cases, this amount of
+storage should be sufficient for thirty days of monitoring data. Organizations
+are responsible for managing the size of the persistent volume.
+
+`retentionPeriod` controls how long usage data is retained. For example, `"120m"`
+for 120 minutes, `"36h"` for 36 hours, `14d` for two weeks, or `"0"` for unbounded
+retention. Units smaller than seconds or larger than days are not currently
+supported. `retentionPeriod` does not apply to other types of data stored by
+Chronicle.
```

Comment:
This feature was recently moved to undocumented (see https://github.com/rstudio/chronicle/pull/2093), so we should probably remove references to it here as well.

Reply:
Removed.